src/Microsoft.AspNet.Mvc.ModelBinding/BinderMetadata/FromServicesAttribute.cs

```diff
@@ -8,8 +8,60 @@ namespace Microsoft.AspNet.Mvc
 {
     /// <summary>
     /// This attribute is used on action parameters or model properties to indicate that
-    /// they will be bound using service provider.
+    /// they will be bound with the <see cref="ServicesModelBinder"/> using the <see cref="IServiceProvider"/>.
     /// </summary>
+    /// <example>  
```

Comment:
I hadn't considered intellisense, I added a couple examples just for reference and/or for MSDN style generated docs. If examples show up in intellisense then there's certainly not much point in putting detailed examples in code comments. Happy to remove these examples or update based on your recommendations and just wait until the markdown docs repo is available.